components/hosted-competition/HostedCompetitionParticipantsContext.js

```diff
@@ -471,12 +480,51 @@ export default class HostedCompetitionParticipantsContext extends BaseFuroContex
       [COMPETITION_PARTICIPANT_STATUS.DISQUALIFIED.ID]: 'text-error',
     }
 
-    return Object.values(COMPETITION_PARTICIPANT_STATUS)
+    return this.competitionParticipantStatuses
       .map(status => ({
-        label: status.NAME,
-        value: String(status.ID),
-        class: classHash[status.ID],
+        label: this.formatCompetitionParticipantStatus({
+          statusName: status.name,
+        }),
+        value: String(status.statusId),
+        class: classHash[status.statusId],
+      }))
+  }
+
+  /**
+   * Format competition participant status.
+   *
+   * @param {{
+   *   statusName: string
+   * }} params - Parameters.
+   * @returns {string}
+   */
+  formatCompetitionParticipantStatus ({
+    statusName,
+  }) {
+    return statusName.replaceAll('_', ' ')
+      .split(' ')
```

Comment:
@hecticme 

Does not it work? 🤔

```suggestion
    return statusName
      .split('_')
```

Reply:
Same. Thank you :)